dhis-2/dhis-test-e2e/src/test/java/org/hisp/dhis/tracker/imports/RuleEngineTests.java

```diff
@@ -171,6 +170,78 @@ public void shouldShowErrorOnCompleteInTrackerEvents() throws Exception {
         .body("message", hasItem(stringContainsInOrder("ERROR ON COMPLETE ")));
   }
 
+  @Test
```

Comment:
Can we add a test to simulate when the user do not have access to the program stage present in the program rule action?